ansible_collections/arista/avd/roles/eos_designs/templates/network_services/router-bgp-vrfs.j2

```diff
@@ -7,9 +7,18 @@
 {%         for vrf in tenant.vrfs %}
 {%             set bgp_vrf.address_family_ipv4_neighbors = [] %}
 {%             set bgp_vrf.address_family_ipv6_neighbors = [] %}
-{%             set bgp_vrf_vni = vrf.vrf_vni | arista.avd.default(vrf.vrf_id) %}
-{%             set leaf_evpn_rt = (network_services_data.evpn_rt_admin_subfield or bgp_vrf_vni) ~ ':' ~ bgp_vrf_vni %}
-{%             set route_targets = namespace(import={'evpn': [leaf_evpn_rt]}, export={'evpn': [leaf_evpn_rt]}) %}
+{%             set bgp_vrf_id = vrf.vrf_vni | arista.avd.default(vrf.vrf_id) %}
+{%             set bgp_overlay_rt_admin_subfield = network_services_data.evpn_rt_admin_subfield %}
```

Comment:
consider renaming the evpn variable to overlay for consistency

Reply:
Fixed in PR https://github.com/aristanetworks/ansible-avd/pull/1488.